src/Moontoast/Math/BigNumber.php

```diff
@@ -647,10 +647,11 @@ public static function convertFromBase10($number, $toBase)
         $outNumber = '';
         
         while (bccomp($number, $toBase) >= 0) {
-            $quotient = bcdiv($number, $toBase, 0);
-            $remainder = bcsub($number, bcmul($quotient, $toBase, 0), 0);
+            list($number, $remainder) = array(
+                bcdiv($number, $toBase, 0),
+                bcmod($number, $toBase)
+            );
```

Comment:
Ok! But I have a suspicion that bcmod () can be return illegal string '0.0000', can we discuss it here?